pkg/registry/core/pod/rest/metrics.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rest
+
+import (
+	"sync"
+
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+const (
+	namespace = "kube_apiserver"
+	subsystem = "pod_logs"
+
+	usageEnforce     = "enforce_tls"
+	usageSkipAllowed = "skip_tls_allowed"
+	usageSkipDenied  = "skip_tls_denied"
+)
+
+var (
+	// podLogsUsage counts and categorizes how the insecure backend skip TLS option is used and allowed.
+	podLogsUsage = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "pods_logs_insecure_backend",
```

Comment:
should we follow the same convention here - add a suffix `total` to the counter to make it `pods_logs_insecure_backen_total` to be consistent?